includes/class-scheduler.php

```diff
@@ -59,6 +60,10 @@ public static function register_schedules() {
 		if ( ! \wp_next_scheduled( 'activitypub_cleanup_followers' ) ) {
 			\wp_schedule_event( time(), 'daily', 'activitypub_cleanup_followers' );
 		}
+
+		if ( ! \wp_next_scheduled( 'activitypub_reprocess_outbox' ) ) {
+			\wp_schedule_event( time(), 'hourly', 'activitypub_reprocess_outbox' );
```

Comment:
Is this still missing a hook to connect the `activitypub_reprocess_outbox` event to the `reprocess_outbox` callback? Or did I miss that?